include/model.php

```diff
@@ -747,28 +769,55 @@ public function get_terms_with_no_lang( $taxonomies, $limit ) {
 	 *
 	 * @since 1.2
 	 * @since 3.4 Moved from PLL_Admin_Model class.
-	 * @since 3.4 Change method signature.
+	 *            Removed `$limit` parameter, added `$lang` and `$types` parameters.
 	 *
-	 * @param int $limit Max number of posts or terms to return. Defaults to -1 (no limit).
+	 * @param PLL_Language|null $lang  Optional. The language to assign to objects. Defaults to `null` (default language).
+	 * @param string[]          $types Optional. Types to handle (@see PLL_Translatable_Object::get_type()). Defaults
+	 *                                 to an empty array (all types).
 	 * @return void
-	 *
-	 * @phpstan-param -1|positive-int $limit
 	 */
-	public function set_language_in_mass( $limit = -1 ) {
-		$nolang = $this->get_objects_with_no_lang( $limit );
+	public function set_language_in_mass( PLL_Language $lang = null, array $types = array() ) {
+		if ( empty( $lang ) ) {
```

Comment:
```suggestion
	public function set_language_in_mass( $lang = null, array $types = array() ) {
		if ( ! $lang instanceof PLL_Language ) {
```
Reasoning: This is a public method. If someone calls it with the legacy param, `set_language_in_mass( 1000 )`, we avoid to trigger a fatal.

Reply:
Done in [acf206e](https://github.com/polylang/polylang/pull/1258/commits/acf206e1effdd2bb39ba2f4ec10c5431c4240fb2).